src/heroes/bastion/reconfigure.opy

```diff
@@ -1,16 +1,25 @@
 #!mainFile "../../dev_main.opy"
 
+playervar sentry_healing
+
 
 rule "[bastion/reconfigure.opy]: Reduce assault form damage and movement speed":
     @Event eachPlayer
     @Hero bastion
     @Condition eventPlayer.isUsingAbility1()
     
     eventPlayer.setDamageReceived(percent(ADJ_BASTION_IRONCLAD_RESISTANCE/OW2_BASTION_IRONCLAD_RESISTANCE))
+    eventPlayer.movement_helper = percent(ADJ_BASTION_RECONFIGURE_MOVESPEED/OW2_BASTION_RECONFIGURE_MOVESPEED)
     eventPlayer.setMoveSpeed(percent(ADJ_BASTION_RECONFIGURE_MOVESPEED/OW2_BASTION_RECONFIGURE_MOVESPEED))
+    eventPlayer.startHoT(eventPlayer, OW2_BASTION_ASSAULT_CONFIG_DURATION, ADJ_BASTION_ASSAULT_CONFIG_HPS)
```

Comment:
design choice comment: bastion starts healing as soon as the transformation animation starts, but I think he should only start healing after the transformation finishes